releases/v22_0_0.rst

```diff
@@ -0,0 +1,255 @@
+Note that not all FilterLabel objects have both a band and a physical filter, especially during the transition period, so please program defensively.
+See `community.lsst.org <https://community.lsst.org/t/migrating-from-afw-image-filter-to-filterlabel/459>`__ for details.
+:jirab:`RFC-730`
+
+.. _release-v22-0-0-remove-obs_ctio0m9:
+
+Removal of the obs_ctio0m9 package
+----------------------------------
+
+The `~obs_ctio0m9` camera package has been removed.
+:jirab:`RFC-729, DM-26867, DM-26868`
+
+
+.. _release-v22-0-0-remove-metric-in-commonMetrics.py:
+
+Removal of metric configurations in commonMetric.py
+---------------------------------------------------
+
+The :code:`metric` field has been replaced by :code:`connections.package` and
+:code:`connections.metric`.
+
+
+.. _release-v22-0-0-pending-deprecations:
+
+Pending Deprecations
+====================
+
+There are no known pending deprecations.
+
+.. _release-v22-0-0-deprecations:
+
+Deprecations
+============
+
+These packages/functions are deprecated and will not be available in the next major release.
+
+- :ref:`release-v22-0-0-deprecate-gen2`
+- :ref:`release-v22-0-0-deprecate-validate-drp`
+- :ref:`release-v22-0-0-deprecate-Filter`
+- :ref:`release-v22-0-0-deprecate-configurations-in-fgcmFitCycle.py`
+- :ref:`release-v22-0-0-deprecate-configurations-in-psfexStarSelector.py`
+
+.. _release-v22-0-0-deprecate-gen2:
+
+Deprecated Generation 2 Middleware
+----------------------------------
+Generation 2 middleware (Gen2) is no longer being developed and should not be used for new code.
+Gen2 infrastructure code (within e.g., :ref:`pipe_tasks <lsst.pipe.tasks>`, :ref:`pipe_base <lsst.pipe.base>`, :ref:`obs_base <lsst.obs.base>`, `daf_persistence <https://github.com:lsst/daf_persistence>`_, and obs packages) will no longer be maintained after January 1, 2022 and may be removed at any point afterwards.
+The CI package `ci_hsc_gen2` will continue to be run, and the tasks it checks will be maintained until we remove the Gen2 infrastructure code in 2022.
+Following this release, we will begin to drop gen2 pipelines from our verification packages (e.g., ap_verify), where a functional and validated gen3 pipeline exists.
+
+.. _release-v22-0-0-deprecate-validate-drp:
+
+Removal of validate_drp
+------------------------
+The algorithms implemented in :ref:`validate_drp <lsst.validate.drp>` were ported as-is to run in :ref:`faro <lsst.faro.base>`.
+All future development of scientific performance metrics will be carried out in faro.
+:ref:`validate_drp <lsst.validate.drp>` will be removed before the release of version 24.0.0.
+
+.. _release-v22-0-0-deprecate-Filter:
+
+Removal of `afw.lsst.Filter`
+----------------------------
+`~afw.lsst.Filter` has been replaced with :ref:`filterLabel<release-v22-0-0-filterLabel>`, marked as deprecated, and will be removed before the release of version 24.0.0.
+:jirab:`RFC-730`
```

Comment:
Double check `afw.lsst.Filter` — do you mean `lsst.afw…`? Also double check the link for Filter in the rendered output. With single backticks it should either resolve into a link to should be turned into double backticks to make it an unlinked code literal.